object_manager_mx_record.go

```diff
@@ -41,47 +45,59 @@ func (objMgr *ObjectManager) GetMXRecordByRef(ref string) (*RecordMX, error) {
 	return recordMX, err
 }
 
-func (objMgr *ObjectManager) GetMXRecord(dnsview string, fqdn string) (*RecordMX, error) {
-	if dnsview == "" || fqdn == "" {
+func (objMgr *ObjectManager) GetMXRecord(dnsView string, fqdn string) (*RecordMX, error) {
+	if dnsView == "" || fqdn == "" {
 		return nil, fmt.Errorf("DNS view and fqdn are required to retrieve a unique mx record")
 	}
 	var res []RecordMX
 
 	recordMX := NewRecordMX(RecordMX{})
 
 	sf := map[string]string{
-		"view": dnsview,
+		"view": dnsView,
 		"name": fqdn,
 	}
 	queryParams := NewQueryParams(false, sf)
 	err := objMgr.connector.GetObject(recordMX, "", queryParams, &res)
 
 	if err != nil {
 		return nil, err
-	} else if res == nil || len(res) == 0 {
-		return nil, NewNotFoundError(
-			fmt.Sprintf(
-				"MX record with name '%s' in DNS view '%s' is not found",
-				fqdn, dnsview))
 	}
 
 	return &res[0], err
```

Comment:
In case of an error, this place cannot be reached. Thus, either the code above (which was removed) should be returned back, or the line should become 'return &res[0], nil'.

Removing the code above makes it harder for a developer to figure out (not obvious) how to distinguish between 'not found' error and other error types in the code, which uses GetMXRecord() method.